Source/SwiftLintFramework/Rules/Performance/SortedFirstLastRule.swift

```diff
@@ -38,7 +38,7 @@ public struct SortedFirstLastRule: CallPairRule, OptInRule, ConfigurationProvide
 
     public func validate(file: File) -> [StyleViolation] {
         return validate(file: file,
-                        pattern: "[\\}\\)]\\s*\\.(first|last)",
+                        pattern: "[\\}\\)]\\s*\\.(first|last)(?!Index)",
```

Comment:
can you add some non-triggering examples to make sure we don't break this in the future?

Reply:
@marcelofabri thanks, I added some examples.